ci/taos/plugins-good/pr-format-doxygen-tag.sh

```diff
@@ -53,6 +53,11 @@ function pr-format-doxygen-tag(){
                     # Append a doxgen rule step by step
                     doxygen_basic_rules="@file @brief" # @file and @brief to inspect file
                     doxygen_advanced_rules="@author @bug" # @author, @bug to inspect file, @brief for to inspect function
+
+                    # When check_result is skip, set value to success(default)
+                    if [[ $check_result == "skip" ]]; then
+                        check_result="success"
+                    else
```

Comment:
It's strange. else or fi? please check the statement correct?

Reply:
fixed. thanks.